src/policy/fees.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
In that case wouldn't it make sense to move `fees.h` to `src/consensus`?

Reply:
That could make sense too. For the sake of maintaining similarity with Bitcoin's code base, it may be better to keep it under "policy" (unless the header dependencies are an issue). People familiar with Bitcoin may look here first.